app/src/main/java/com/douglasbruce/tasky/core/data/database/model/ModifiedReminderEntity.kt

```diff
@@ -0,0 +1,13 @@
+package com.douglasbruce.tasky.core.data.database.model
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import com.douglasbruce.tasky.core.model.ModificationType
+
+@Entity(
+    tableName = "modified_reminders"
+)
+data class ModifiedReminderEntity(
+    @PrimaryKey(autoGenerate = false) val reminderId: String,
+    val type: ModificationType,
+)
```

Comment:
I think you don't need a separate table for all of these. If you make the modification type variable, you can just as well make the agenda item type variable, so you only have one table `ModifiedAgendaItemEntity` which contains the fields `id`, `modificationType` and `agendaItemType`